MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -98,7 +88,7 @@ import Foundation
             switch controllerResponse.result {
             case .success(let accessToken):
```

Comment:
can you rename this variable to "accessTokenResult"?

Reply:
Done